heat_pump_adoption_modelling/pipeline/MCS/mcs_epc_joining.py

```diff
@@ -0,0 +1,322 @@
+# File: heat_pump_adoption_modelling/pipeline/MCS/mcs_epc_joining.py
```

Comment:
Would it make sense to move this to `heat_pump_adoption_modelling/pipeline/preprocessing` or create a new subfolder in pipeline called `data_merging/data_joining`?

Reply:
Good idea - have moved to /preprocessing for now